src/main/java/org/primefaces/component/timeline/TimelineBase.java

```diff
@@ -172,21 +182,29 @@ public void setBrowserTimeZone(Object browserTimeZone) {
     }
 
     public String getHeight() {
-        return (String) getStateHelper().eval(PropertyKeys.height, "auto");
+        return (String) getStateHelper().eval(PropertyKeys.height, null);
```

Comment:
Is "auto" no longer a valid option in the new timeline?  What are the options for height?

Reply:
No, "auto" is no longer a valid options, that was something that give me an error on tests. If the property is omitted it behaves like auto (see next image of description of height property). 

![image](https://user-images.githubusercontent.com/16229253/64270195-ca26c980-cf00-11e9-9d17-232c2e54e3a4.png)